src/Slices/ResourceV2/UI/resourceDetailsTab.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useContext } from "react";
+import { Flex, FlexItem } from "@patternfly/react-core";
+import styled from "styled-components";
+import { Query, RemoteData } from "@/Core";
+import { useUrlStateWithString } from "@/Data";
+import { TabKey, Tabs } from "@/Slices/ResourceDetails/UI/Tabs";
+import { DependencyContext, words } from "@/UI";
+import {
+  Description,
+  PageContainer,
+  ResourceStatusLabel,
+} from "@/UI/Components";
+
+interface ResourceTabsProps {
+  resourceId: string;
+}
+
+export const ResourceDetailsTab: React.FunctionComponent<ResourceTabsProps> = ({
+  resourceId,
+}) => {
+  const { queryResolver } = useContext(DependencyContext);
+  const [activeTab, setActiveTab] = useUrlStateWithString<TabKey>({
+    default: TabKey.Attributes,
+    key: `tab`,
```

Comment:
It's ok for now, but the key is not unique on the page, so if you change the active tab at one place you will change it in every row.

Reply:
I see, this I need to remember to change in the future :p